### PR TITLE
Added argo workflow and oc template

### PR DESCRIPTION
**Pull request by saisankargochhayat**
OC Template and the Argo workflow template. 
Test run using oc template - http://argo-ui-thoth-test-core.cloud.paas.psi.redhat.com/workflows/thoth-test-core/kebechet-job-122

### CODE REVIEW

File: workflows/templates/kebechet-job.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          - name: KEBECHET_SUBCOMMAND
+            value: "run-webhook"
+          - name: KEBECHET_PAYLOAD
+            value: "{{inputs.parameters.WEBHOOK_PAYLOAD}}"
+          - name: GITHUB_KEBECHET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: github-oauth-token
+                name: kebechet-secret
+          - name: GITLAB_KEBECHET_TOKEN
```
Comment (harshad16) [inline workflows/templates/kebechet-job.yaml:45]: @saisankargochhayat how do you wanna add the Gitlab cee ?
will this name is fine for gitlab cee variable ?
https://github.com/thoth-station/thoth-ops/blob/b3be9200456ce704e64d2cb7aa6eaf9b53764b9a/ansible/prepare_environment.yaml#L381

File: workflows/templates/kebechet-job.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+spec:
+  templates:
+    - name: kebechet-job
+      inputs:
+        parameters:
+        - name: WEBHOOK_PAYLOAD 
+      container:
+        # Chnage to thoth station kebechet master, repo.
+        image: "quay.io/saisankargochhayat/kebechet:master"
```
Comment (harshad16) [inline workflows/templates/kebechet-job.yaml:20]: can you update the thoth-station kebechet image here.
https://quay.io/repository/thoth-station/kebechet-job?tab=tags

**Comment by pacospace** [comment]
Depends-On: https://github.com/thoth-station/common/pull/780

File: workflows/templates/kebechet-job.yaml (hunk 2 above)
Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml:20]: @harshad16 I dont seem to have access to it. Can you grant me access to the repo on quay?

File: workflows/templates/kebechet-job.yaml (hunk 1 above)
Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml:45]: @harshad16 I will test a the gitlab cee part later and add it along with the #336 pr. :smile:

File: workflows/templates/kebechet-job.yaml (hunk 2 above)
Comment (harshad16) [reply workflows/templates/kebechet-job.yaml:20]: image would be pushed by thoth-ops.
` image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/kebechet:{{inputs.parameters.IMAGE_STREAM_TAG}}"`

Comment (harshad16) [reply workflows/templates/kebechet-job.yaml:20]: Please have a look at the example from https://github.com/thoth-station/adviser/blob/170e0d2e4d46a055010d03a5b58c643664e3c06c/openshift/adviser-template.yaml#L99

Comment (harshad16) [reply workflows/templates/kebechet-job.yaml:20]: @pacospace @fridex WDYT about this ?

Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml]: I have updated it to use image from openshift `thoth-test-core`

File: openshift/kebechet-template.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# This template uses the kebechet-job argo workflow to run kebechet on a particular webhook. 
+apiVersion: v1
+kind: Template
+metadata:
+  name: kebechet-job
```
Comment (fridex) [inline openshift/kebechet-template.yaml:5]: Just kebechet? I think there will be no "job". Here and at other places as well.

File: workflows/templates/kebechet-job.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      container:
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/\
+        {{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/\
+        kebechet-job:{{inputs.parameters.IMAGE_STREAM_TAG}}"
+        command: [sh, -c]
+        args: [
+            '
+            ./app.sh
+            ',
+          ]
```
Comment (fridex) [inline workflows/templates/kebechet-job.yaml:30]: Is this needed? I think the Dockerfile already states this (and we should slowly migrate to s2i if possible).

File: openshift/kebechet-template.yaml (hunk 3 above)
Comment (saisankargochhayat) [reply openshift/kebechet-template.yaml:5]: I will change it :)

File: workflows/templates/kebechet-job.yaml (hunk 4 above)
Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml:30]: Umm it seems to not trigger the job, unless this is stated, I can try again.

Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml]: I have removed it now.

Comment (fridex) [reply workflows/templates/kebechet-job.yaml]: Does it work even if removed?

Comment (saisankargochhayat) [reply workflows/templates/kebechet-job.yaml]: @fridex It triggers the job.

**Pull request merged**